### ISSUE TITLE
Feature request: Option to sort working column

### ISSUE THREAD

**Issue by signallhill** [cantino/huginn#1253]
Request option to be able to sort working? column so that all no comes on top.

**Comment by dsander** [comment]
I am afraid that would require a lot of work, the `working` column is not present in the database but implemented by every agent in ruby. To make it work we would need to change every agent to either add a cached `working` attribute or implement the logic in SQL. We could sort the current page via javascript but that could be confusing because the other sorting options done for all pages.

**Comment by cantino** [comment]
Agreed.  We'd need to add a job that checks `working?` on each Agent periodically and caches it in the DB.

**Comment by virtadpt** [comment]
Maybe in an instance of MySQL's MEMORY storage engine?  There's no sense in writing that data to disk because it's so volatile.

**Comment by sfischer13** [comment]
I was looking for that feature as well, because it would make trouble-shooting easier.
